### PR TITLE
Optimize PF_IsMapValid_I

### PR DESCRIPTION
**Pull request by In-line**

### CODE REVIEW

File: rehlds/engine/pr_cmds.cpp [hunk 1]
```diff
@@ -1541,11 +1541,15 @@ int EXT_FUNC PF_precache_generic_I(const char *s)
 
 int EXT_FUNC PF_IsMapValid_I(const char *mapname)
 {
+#ifdef REHLDS_FIXES
+	char cBuf[42];
```
Comment (theAsmodai) [inline rehlds/engine/pr_cmds.cpp]: :suspect:

Comment (WPMGPRoSToTeMa) [inline rehlds/engine/pr_cmds.cpp]: Need a comment like:
`[len('maps/')=5] + 32 + [len('.bsp')=4] + [eos=1]`

Comment (theAsmodai) [inline rehlds/engine/pr_cmds.cpp]: It's a very minor optimization. Use of 64 would been easier than to count characters.

Comment (WPMGPRoSToTeMa) [inline rehlds/engine/pr_cmds.cpp]: @theAsmodai yup.

**Pull request merged**